resip/stack/DateCategory.cxx

```diff
@@ -25,20 +25,34 @@ namespace resip
 struct days { const char *name; DayOfWeek type; };
 class DayOfWeekHash
 {
+#if  (GPERF_MAJOR_VER > 3 || (GPERF_MAJOR_VER == 3 && GPERF_MINOR_VER >= 1))
```

Comment:
As part of configuration, could you please add an `GPERF_SIZE_TYPE` and set it to `unsigned int` for older and `size_t` for newer gperf? Then use the `GPERF_SIZE_TYPE` here without ifdefs.